### ISSUE TITLE
[Feature request] Frontend behind a reverse proxy

### ISSUE THREAD

**Issue by tr11** [metrico/gigapipe#414]
It'd be great to have an environment variable to allow serving the UI under a different base url so it plays better with nginx or some other reverse proxy.

**Comment by lmangani** [comment]
@tr11 good point, we might add this to the options but this it technically already possible:  `qryn-view` is completely independent from qryn itself and only bundled for extra joy of demo & testing.  The webapp is static and only a client to the API so you can serve it from any service of your preference, or even directly by nginx 😉

**Comment by tr11** [comment]
Thanks for the quick reply. We are already doing exactly that by building the view project with the subpath we need. It's more for convenience having everything in the same docker image.

**Comment by akvlad** [comment]
@tr11 a bit of clarification:
do you want a special configuration like `ADVANCED_UI_BASE=/view` for the bundled view to open in `http://.../view` ,?

**Comment by tr11** [comment]
@akvlad, yes, that would be perfect. That would allow us to serve the UI easily under the same `monitoring`  subpaths we use for other tools!
